docker scripts/unix/change-user-role.sh

```diff
@@ -0,0 +1,11 @@
+echo "available roles:"
+echo -e "\trole_worker"
+echo -e "\trole_manager"
+echo -e "\trole_admin"
+read -p "new role: role_" role
+read -p "email: " username
+echo "--DB password"
+docker exec -it skynest-db mysql -u user -p --execute="USE skynest; CALL change_user_role('$username','role_$role')"
```

Comment:
So you want to create a script that will be used to change the role dynamically on whatever user you like?

Reply:
Initially, I just wanted a script to promote existing users to admins. Then I realised that I should probably have one to demote admins as well. In the end, it turned into something completely generic.